tests/test_bears/TestBear.py

```diff
@@ -1,8 +1,20 @@
 from coalib.bears.LocalBear import LocalBear
+from coalib.bearlib.aspects import map_setting_to_aspect
+from coalib.bearlib.aspects.Formatting import TrailingSpace
 
 
-class TestBear(LocalBear):
+class TestBear(
```

Comment:
It is better to make a new `AspectsTestBear` instead of changing the `TestBear` IMO.

Reply:
But we need tests to ensure the old bear system does not break.
Please create a subclass `AspectsTestBear`

Reply:
Ok I will, in `BearTest.py` only or make a new py file?

Reply:
Now it's being tested using `AspectsGeneralTestBear` created in the `languages=all` patch.